fritzconnection/cli/fritzwol.py

```diff
@@ -0,0 +1,73 @@
+"""
+fritzwol.py
+
+Module to wake up a single host via the Fritzbox built-in mechanism.
+This can be helpful if the host to be woken up is in a different
+broadcast domain/ subnet than the client which tries to wake up.
+CLI interface.
+
+This module is part of the FritzConnection package.
+https://github.com/kbr/fritzconnection
+License: MIT (https://opensource.org/licenses/MIT)
+Author: Maik Töpfer
+"""
+
+from fritzconnection.core.exceptions import FritzConnectionException
+from fritzconnection.core.fritzconnection import FritzConnection
+from ..lib.fritzhosts import FritzHosts
+from . utils import get_cli_arguments, get_instance, print_header
+
+
+class WakeOnLan:
```

Comment:
Please add a doc-string the class and all methods. readthedocs uses this to build the online-documentation.